src/ruby/spec/generic/rpc_server_spec.rb

```diff
@@ -95,7 +95,7 @@ class FailingService
   def initialize(_default_var = 'ignored')
     @details = 'app error'
     @code = 101
-    @md = { failed_method: 'an_rpc' }
+    @md = { 'failed_method' => 'an_rpc' }
```

Comment:
Do hash keys sent from one end show up only as a hash key of that type on the other end?